extensions/amp-story/1.0/amp-story-tooltip.js

```diff
@@ -358,10 +428,22 @@ export class AmpStoryTooltip {
     if (!closest(dev().assertElement(event.target),
         el => el == this.tooltip_)) {
       event.stopPropagation();
+      this.clearTooltip_();
       this.closeTooltip_();
     }
   }
 
+  /**
+   * Clears any attributes or handlers that may have been added to the tooltip,
+   * but weren't used.
```

Comment:
Can you explain why this might happen?

Reply:
Added comment. The case when it might happen is when e.g. when user clicks on an expandable target but then clicks away.